File: Assignment 17/Question4.java
import java.util.Scanner;

class Logic
{
    
    public int CheckMax(int iNo1 ,int  iNo2, int iNo3) 
    {
        if (iNo1<iNo2 && iNo1<iNo3)
        {
            return iNo1;
        }

        else if(iNo2<iNo1 && iNo2<iNo3)
        {
            return iNo2;
        }
        else
        {
            return iNo3;
        }
        
    
        
       
    }
}          
class Question4
{
    public static void main(String[] args) 
    {
        
        Scanner sobj = new Scanner(System.in);
        int iValue1 = 0 ;
        int iValue2 = 0 ;
        int iValue3 = 0 ;
        int iRet = 0 ;
     

        System.out.println("Enter your Number :");
        iValue1 = sobj.nextInt();

        System.out.println("Enter your Number :");
        iValue2 = sobj.nextInt();

        System.out.println("Enter your Number :");
        iValue3 = sobj.nextInt();

        Logic lobj = new Logic();

        iRet = lobj.CheckMax(iValue1,iValue2,iValue3);

       
        
          System.out.println("The minimum no is : " + iRet);
        
    }    
}

File: Assignment 11/Question4.c
 #include <stdio.h>

int RangeEvenAdd(int iNo1 , int iNo2)
{
 
    int iCnt = 0 ;
    int iAdd = 0 ;
   
    if (iNo1>iNo2 || iNo1<0 || iNo2<0 )
    {
        return -1;     
    }

    for (iCnt = iNo1 ; iCnt<=iNo2 ; iCnt++)
    {
        if(iCnt % 2 == 0)
        {
            iAdd = iAdd + iCnt;
        } 
    }
    return iAdd ;
}
int main() 
{
   int iValue1 = 0 , iValue2 = 0 , iRet = 0;

   printf("Enter first number :");
   scanf("%d",&iValue1);

   printf("Enter second number :");
   scanf("%d",&iValue2);

   iRet = RangeEvenAdd(iValue1,iValue2);

   if(iRet<0)
    {
        printf("Invalid Range");
    }
    else
    {
    printf("%d is the addition of the range",iRet);
    }

    return 0 ;
}
